src/components/Lesson/lesson.helpers.ts

```diff
@@ -327,3 +331,35 @@ export function groupLessonPathways(pathways: LessonPathway[]) {
     subjects,
   };
 }
+
+type Attribution = {
+  questionNumber: string;
+  attribution: string;
+};
+
+export const createAttributionObject = (
+  questions: LessonOverviewQuizData,
+): Attribution[] | [] => {
+  const attributions: Attribution[] = [];
+  if (questions) {
+    questions.forEach((question, index) => {
```

Comment:
this should be a reduce as we're constructing a new value from an array - the inner forEach should be a filter&map